blockprotocol/blockprotocol
# Display Repo Link on Blocks

## What this Does

Following up from https://github.com/hashintel/hash/pull/267, this PR uses the repository data provided with each block to display the data on the block pages on the blockprotocol website.

## What does this change

- Updates the timestamp on each block to fetch the latest block data [like here](https://github.com/blockprotocol/blockprotocol/pull/198/files#diff-79bb22f55dc74a87db2779ddba16893b31a18095e96f3ef1c46d9a7a479ae837R6).
- Display the repo data on the block pages - [link](https://github.com/blockprotocol/blockprotocol/pull/198/files#diff-f7e07a326bf27cecf1b90c733abb2b030eeec66409cd87086bdf1f265565f301R292).
- Modify the `readBlocks` code to read and convert the repository to the current format at runtime - [link](https://github.com/blockprotocol/blockprotocol/pull/198/files#diff-ef6b34f7ca0be2a76177cfb93e2f30a49546f3becd1311ed3a4815adc260abb2R75)

## How to test this

- Build the blocks locally
- Visit the block pages to check if the repo link appears

![chrome_Gwt8zwX4ke](https://user-images.githubusercontent.com/22196279/152768084-9683ec6a-2b99-4400-a7b6-fe57c9664d5d.png)
![chrome_4dUITRZ78G](https://user-images.githubusercontent.com/22196279/152768093-d71dee2b-b4c1-4059-96a5-08fabeb89614.png)



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201540186206731
